test/new_relic/agent/method_tracer_test.rb

```diff
@@ -302,12 +291,12 @@ def test_trace_method_with_block
   def test_trace_module_method
     NewRelic::Agent.add_method_tracer :module_method_to_be_traced, '#{args[0]}'
```

Comment:
There is an interpolated string used as the metric name for this test as well, does this need to be updated to use a proc?

Reply:
My eyes glazed over this particular test because it's not doing anything interesting. It's just testing that tracing a module method works without error, which we're already basically doing below in `test_add_class_method_tracer`. I think I'll rewrite this test to work more like that one.